pipelines/nextflow/workflows/nextflow.config

```diff
@@ -17,51 +17,98 @@ profiles {
 
     standard {
         process {
-            executor = 'lsf'
-            queue = { task.memory < 300.GB ? 'standard' : 'bigmem' }
+            executor = 'local'
```

Comment:
We mentioned to use `local` for testing. Would it be also useful to add the scope for `report` after the `process scope`? E.g.
```
process {
}
report {
    enabled = true
    overwrite = true
}
```

Reply:
Maybe we want that report systematically? So for all profiles?

Reply:
Topic for next Tuesday then :+1: